spec/requests/public_spec.rb

```diff
@@ -26,7 +24,7 @@ def update_page(slug, text)
     )
   end
 
-  it 'should render the CMS page' do
+  it 'renders the CMS page' do
```

Comment:
Do these tests write out which CMS page they are validating? Its also likely that this should be an array of pages and expected responses. As we add CMS pages, we don't want the tests to balloon out

Reply:
These aren't new tests.

This test in particular just makes sure that markdown is rendered and accessible by an endpoint.

All the CMS requests go through a single controller, so if one request works as expected, then the rest will too